lerna.json

```diff
@@ -5,7 +5,9 @@
     "example",
     "example/lambdas/*",
     "example/scripts/*",
-    "lambdas/*",
+    "lambdas/db-migration",
```

Comment:
Nice work.    We should write a p3 ticket or comment in the merge ticket if one exists that this needs sorted.  We need to restore this file on main merge as well.

Reply:
![image](https://user-images.githubusercontent.com/90708967/167920167-89d48725-31d0-407a-ade8-b677e07cf11c.png)

Actionable P3 ticket has been created as part of rds-phase-3 epic
https://bugs.earthdata.nasa.gov/browse/CUMULUS-2946